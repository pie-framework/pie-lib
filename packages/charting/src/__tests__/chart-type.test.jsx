import { shallow } from 'enzyme';
import React from 'react';
import ChartType from '../chart-type';

describe('ChartType', () => {
  let wrapper;
  let props;
  const onChange = jest.fn();

  beforeEach(() => {
    props = {
      classes: {},
      value: 'bar',
      onChange,
    };

    wrapper = (newProps) => {
      const configureProps = { ...props, newProps };

      return shallow(<ChartType {...configureProps} />);
    };
  });

  describe('renders', () => {
    it('snapshot', () => {
      expect(wrapper()).toMatchSnapshot();
    });
  });
});
