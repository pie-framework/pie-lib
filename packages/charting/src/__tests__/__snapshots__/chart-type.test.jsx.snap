// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChartType renders snapshot 1`] = `
<Component
  classes={
    Object {
      "chartType": "Component-chartType-1",
      "chartTypeLabel": "Component-chartTypeLabel-2",
    }
  }
  onChange={[MockFunction]}
  value="bar"
/>
`;
